Lesson5/Task1/RationalNumber.cs

```diff
@@ -0,0 +1,194 @@
+﻿public class RationalNumber : ICloneable
+{
+    /// <summary>
+    /// числитель
+    /// </summary>
+    private int _numerator = 0;
+    /// <summary>
+    /// знаменатель
+    /// </summary>
+    private int _denominator = 1;
```

Comment:
```suggestion
    private readonly int _numerator = 0;
    /// <summary>
    /// знаменатель
    /// </summary>
    private readonly int _denominator = 1;
```
Неизменяемой структурой